{
  "uid" : "d1fafbeb922ba412",
  "name" : "Failure login to D3O with invalid password",
  "fullName" : "D3O Login: Failure login to D3O with invalid password",
  "historyId" : "f389c2653a05dcf25b1d23626b158805",
  "time" : {
    "start" : 1630844007662,
    "stop" : 1630844016457,
    "duration" : 8795
  },
  "status" : "broken",
  "statusMessage" : "NameError: name 'allure' is not defined\n",
  "statusTrace" : "  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\d3aloginsteps.py\", line 66, in assert_login_unsuccessful\n    allure.attach(context.driver.get_screenshot_as_png(),name=\"assert_login_unsuccessful_error_message_invalid_password\",attachment_type=AttachmentType.PNG)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "NameError: name 'allure' is not defined\n",
    "statusTrace" : "  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\d3aloginsteps.py\", line 66, in assert_login_unsuccessful\n    allure.attach(context.driver.get_screenshot_as_png(),name=\"assert_login_unsuccessful_error_message_invalid_password\",attachment_type=AttachmentType.PNG)\n",
    "steps" : [ {
      "name" : "Given the user launches Chrome Browser",
      "time" : {
        "start" : 1630844007664,
        "stop" : 1630844009342,
        "duration" : 1678
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "When the user opens D3a login page url \"https://www.d3a.io/login\"",
      "time" : {
        "start" : 1630844009343,
        "stop" : 1630844011813,
        "duration" : 2470
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "When the user enters username \"laks.kswamy@gmail.com\" and password \"wrongPassword\"",
      "time" : {
        "start" : 1630844011813,
        "stop" : 1630844012155,
        "duration" : 342
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "And the user clicks the Login Button",
      "time" : {
        "start" : 1630844012156,
        "stop" : 1630844016255,
        "duration" : 4099
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "Then the home page does not open with invalid password",
      "time" : {
        "start" : 1630844016255,
        "stop" : 1630844016457,
        "duration" : 202
      },
      "status" : "broken",
      "statusMessage" : "NameError: name 'allure' is not defined\n",
      "statusTrace" : "  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lakshmi krishnaswamy\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\d3aloginsteps.py\", line 66, in assert_login_unsuccessful\n    allure.attach(context.driver.get_screenshot_as_png(),name=\"assert_login_unsuccessful_error_message_invalid_password\",attachment_type=AttachmentType.PNG)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : true,
      "attachmentsCount" : 0,
      "hasContent" : true,
      "stepsCount" : 0
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : true,
    "stepsCount" : 5
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "D3O Login"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "d1fafbeb922ba412.json",
  "parameterValues" : [ ]
}